### PR TITLE
Timer formatting fixes

### PR DESCRIPTION
**Pull request by keszybz**
Follow-up for #13062.

### CODE REVIEW

**Comment by yuwata** [comment]
```
../src/core/dbus-timer.c:296:70: error: unused variable 'c' [-Werror,-Wunused-variable]
                        _cleanup_(calendar_spec_freep) CalendarSpec *c = NULL;
                                                                     ^
1 error generated.
```

File: src/systemctl/systemctl.c [hunk 1]
```diff
@@ -5033,11 +5033,13 @@ static int print_property(const char *name, const char *expected_value, sd_bus_m
...
 
                         while ((r = sd_bus_message_read(m, "(stt)", &base, &v, &next_elapse)) > 0) {
-                                char timespan1[FORMAT_TIMESPAN_MAX], timespan2[FORMAT_TIMESPAN_MAX];
+                                char timespan1[FORMAT_TIMESPAN_MAX], timespan2[FORMAT_TIMESPAN_MAX] = "n/a";
 
-                                bus_print_property_valuef(name, expected_value, value, "{ %s=%s ; next_elapse=%s }", base,
-                                                          format_timespan(timespan1, sizeof(timespan1), v, 0),
-                                                          format_timespan(timespan2, sizeof(timespan2), next_elapse, 0));
+                                format_timespan(timespan1, sizeof timespan1, v, 0);
+                                format_timespan(timespan2, sizeof timespan2, next_elapse, 0);
```
Comment (poettering) [inline src/systemctl/systemctl.c]: maybe cast these to (void)?

File: src/systemctl/systemctl.c [hunk 2]
```diff
@@ -5057,10 +5059,11 @@ static int print_property(const char *name, const char *expected_value, sd_bus_m
                                 return bus_log_parse_error(r);
 
                         while ((r = sd_bus_message_read(m, "(sst)", &base, &spec, &next_elapse)) > 0) {
-                                char timestamp[FORMAT_TIMESTAMP_MAX];
+                                char timestamp[FORMAT_TIMESTAMP_MAX] = "n/a";
 
-                                bus_print_property_valuef(name, expected_value, value, "{ %s=%s ; next_elapse=%s }", base, spec,
-                                                          format_timestamp(timestamp, sizeof(timestamp), next_elapse));
+                                format_timestamp(timestamp, sizeof(timestamp), next_elapse);
```
Comment (poettering) [inline src/systemctl/systemctl.c]: ditto

File: src/core/dbus-timer.c [hunk 3]
```diff
@@ -137,6 +137,38 @@ const sd_bus_vtable bus_timer_vtable[] = {
...
 
+static int timer_add_one_monotonic_spec(
+                Timer *t,
+                const char *name,
+                TimerBase base,
+                UnitWriteFlags flags,
+                usec_t usec,
+                sd_bus_error *error) {
+
+        if (!UNIT_WRITE_FLAGS_NOOP(flags)) {
```
Comment (poettering) [inline src/core/dbus-timer.c:148]: I'd avoid the deep indentation here, and simple exit early here if UNIT_WRITE_FLAGS_NOOP holds

Comment (keszybz) [reply src/core/dbus-timer.c:148]: Yeah, I thought about that, but there's `return 1` at the end, and I prefer to have that once.

**Comment by keszybz** [comment]
Updated with the suggested fixes (except the one about returning early).

**Pull request merged**